src/main/java/duke/Ui/Ui.java

```diff
@@ -0,0 +1,111 @@
+package duke.Ui;
+
+import duke.DukeException.DukeException;
+import duke.Task.Task;
+import duke.TaskList.TaskList;
+
+/**
+ * Collection of the Ui
+ */
+public class Ui {
+    private final String Border = "____________________________________________________________";
```

Comment:
```suggestion
    private final String BORDER = "____________________________________________________________";
```
constant names must be all uppercase